src/main/java/org/commcare/formplayer/objects/SerializableMenuSession.java

```diff
@@ -1,30 +1,61 @@
-package org.commcare.formplayer.repo;
+package org.commcare.formplayer.objects;
 
 import org.commcare.formplayer.session.MenuSession;
+import org.hibernate.annotations.GenericGenerator;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+
+import javax.persistence.*;
 
 /**
  * Created by willpride on 8/1/16.
  */
+
+@Entity
+@Table(name="menu_sessions")
+@EntityListeners(AuditingEntityListener.class)
 public class SerializableMenuSession {
 
+    @Id @GeneratedValue(generator="system-uuid")
+    @GenericGenerator(name="system-uuid", strategy = "uuid")
     private String uuid;
+
+    @Column(name="username")
     private String username;
+
+    @Column(name="domain")
     private String domain;
+
+    @Column(name="appId")
     private String appId;
+
+    @Column(name="installReference")
```

Comment:
change to match what's in the DB:
```suggestion
    @Column(name="installreference")
```